CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 23.06.1+ (???)
 ------------------------------------------------------------------------
+- Feature: [#2013] Add headers for base/clearance height, direction, and ghost flag to the tile inspector.
```

Comment:
i'm guessing this was just forgotten from #2013?

Reply:
That's correct.